src/Propel/Generator/Command/InitCommand.php

```diff
@@ -70,7 +70,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $consoleHelper->writeln('');
 
+        $connectionAttemptLimit = 10;
+        $connectionAttemptCount = 0;
```

Comment:
Previously tests would end up in an infinite loop.

This is an interactive console command that asks the same questions again if a database cannot be connected to. I felt like 10 was a fine and acceptable limit for real humans trying to manually type database connection strings into console prompts.